nixnet/types.py

```diff
@@ -388,7 +391,115 @@ def from_raw(cls, frame):
             constants.FrameType.CANFD_DATA: CanFrame,
             constants.FrameType.CANFDBRS_DATA: CanFrame,
             constants.FrameType.CAN_REMOTE: CanFrame,
+            constants.FrameType.LIN_DATA: LinFrame,
+            constants.FrameType.LIN_BUS_ERROR: LinFrame,
```

Comment:
Looking at the documentation for `LIN_BUS_ERROR`, it seems like we should create a custom frame for it.
http://zone.ni.com/reference/en-XX/help/372841L-01/nixnet/specialframes_c/

Like CAN, it might be reasonable to defer it.